scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl

```diff
@@ -0,0 +1,547 @@
+    String updated_input_files = if (defined(service_account_json_path)) then basename(inputFileofFileNames) else inputFileofFileNames
+
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+          gsutil cp ~{service_account_json_path} local.service_account.json
+          export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
+          gcloud auth activate-service-account --key-file=local.service_account.json
+
+          gsutil cp ~{inputFileofFileNames} ~{updated_input_files}
+       fi
+
+    >>>
+
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_20211101"
+        memory: "1 GB"
+        preemptible: 3
+        cpu: "1"
+        disks: "local-disk 100 HDD"
+    }
+    # ------------------------------------------------
+    # Outputs:
+    output {
+        Array[File] input_jsons = read_lines(updated_input_files)
+    }
+}
+
+task PrepAnnotationJson {
+    input {
+        File annotation_json
+        String output_file_suffix
+        String output_path
+        String? service_account_json_path
+    }
+    parameter_meta {
+        annotation_json: {
+          localization_optional: true
+        }
+    }
+    String output_vt_json = "vat_vt_bq_load" + output_file_suffix
+    String output_genes_json = "vat_genes_bq_load" + output_file_suffix
+    String output_vt_gcp_path = output_path + 'vt/'
+    String output_genes_gcp_path = output_path + 'genes/'
+
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+    String updated_annotation_json = if (defined(service_account_json_path)) then basename(annotation_json) else annotation_json
+
+
+
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+
+            gsutil cp ~{annotation_json} ~{updated_annotation_json}
+        fi
+
+        ## the annotation jsons are split into the specific VAT schema
+        python3 /app/create_variant_annotation_table.py \
+          --annotated_json ~{updated_annotation_json} \
+          --output_vt_json ~{output_vt_json} \
+          --output_genes_json ~{output_genes_json}
+
+        gsutil cp ~{output_vt_json} '~{output_vt_gcp_path}'
+        gsutil cp ~{output_genes_json} '~{output_genes_gcp_path}'
+
+     >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_2022_04_25"
+        memory: "8 GB"
+        preemptible: 5
+        cpu: "1"
+        disks: "local-disk 250 SSD"
```

Comment:
Are you sure you need SSD here?

Reply:
No---I'd ideally like to parameterize a bunch of these values